specs/language/statements.tex

```diff
@@ -1,2 +1,74 @@
 \Ch{Statements}{Stmt}
 \Sec{Label Statements}{Stmt.Label}
+\Sec{Attributes}{Stmt.Attr}
+\Sub{Unroll Attribute}{Stmt.Attr.Unroll}
+The Attribute \textbf{[unroll]} and \textbf{[unroll(n)]} attribute qualifiers can be used to specify that a loop (\textbf{for}, \textbf{while} and \textbf{do while} loops) can be unrolled. This attribute qualifier can be used to specify full unrolling or partial unrolling by a specified amount. This is a compiler hint and the compiler may ignore this directive.
```

Comment:
Other parts of the spec (eg basic.tex) use `\texttt` for code / keywords.

Reply:
It might be my latex setup but I don't see a huge difference with italics. Its very subtle.

Italics:
![image](https://github.com/microsoft/hlsl-specs/assets/1802579/b332e2e6-b801-431d-8312-aa4767195810)

No Italics:
![image](https://github.com/microsoft/hlsl-specs/assets/1802579/87e56db1-5b95-4f4b-903d-5272300e3ec0)

In this example all it did was elongate

Reply:
IIRC, `\texttt` changes the font to the monospaced "typewriter" one.  `\textit` is italics.

Reply:
oh ok  that makes more sense why the change is so subtle.